Softeq.XToolkit.PushNotifications/PushTokenSynchronizer.cs

```diff
@@ -130,14 +130,11 @@ await UnregisterFromRemotePushNotificationsAsync()
         private async Task OnRegisterSuccessInternalAsync(string token)
         {
             IsTokenRegisteredInSystem = true;
+            IsTokenSavedOnServer = false;
```

Comment:
I understand that `OnRegisterSuccessInternalAsync` is only called when the token has changed, but it would probably be more obvious to do `IsTokenSavedOnServer = false` in the `OnTokenChanged` method before calling the `OnRegisterSuccessInternalAsync` method

Reply:
I'm not sure about that. `IsTokenSavedOnServer` relates to the specific token which is stored in `_pushTokenStorageService`. Once we change `_pushTokenStorageService.PushToken`, we immediately mark that this specific token is not saved on server, so it's better to keep these two lines together.
`IsTokenRegisteredInSystem` is also set to true here, so it looks consistent for me.